index.html

```diff
@@ -39,28 +39,28 @@
 						<div class="navbar-end" style="font-weight: 900;">
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light" style="color: #696969">
+									<a class="button is-light" id="navBarSelect">
 										Formatter
 									</a>
 								</div>
 							</div>
 							<div class="navbar-item">
 								<div class="buttons">
-									<a class="button is-light" style="color: #696969">
+									<a class="button is-light" id="navBarSelect">
```

Comment:
Okay nice, disregard previous comment, but make this into a class instead. IDs are for one off elements.